mindjunkies/lecture/models.py

```
@@ -68,3 +69,14 @@ class LectureVideo(BaseModel):
         return str(self.video_title)
+
+
+
+
+class LectureCompletion(BaseModel):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    lecture = models.ForeignKey(Lecture, on_delete=models.CASCADE)
+    completed_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        unique_together = ('user', 'lecture')
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)